src/components/Game.js

```diff
@@ -48,45 +48,46 @@ const Game = () => {
     setXIsNext(!xIsNext);
   };
 
-  const jumpTo = step => {
-    setStepNumber(step);
-    setXIsNext(step % 2 === 0);
-  };
+  // const jumpTo = step => {
```

Comment:
You can just remove this, it's in version control now, so if you need it again, you can just grab it!